Source/WTF/wtf/CagedPtr.h

```diff
@@ -50,36 +48,31 @@ class CagedPtr {
 
     CagedPtr() : CagedPtr(nullptr) { }
     CagedPtr(std::nullptr_t)
-        : m_ptr(shouldTag ? tagArrayPtr<T>(nullptr, 0) : nullptr)
+        : m_ptr(nullptr)
     { }
 
-    CagedPtr(T* ptr, size_t size)
-        : m_ptr(shouldTag ? tagArrayPtr(ptr, size) : ptr)
+    CagedPtr(T* ptr, size_t)
+        : m_ptr(ptr)
     { }
 
-    T* get(size_t size) const
+    T* get(size_t) const
     {
         ASSERT(m_ptr);
         T* ptr = PtrTraits::unwrap(m_ptr);
-        T* cagedPtr = Gigacage::caged(kind, ptr);
-        T* untaggedPtr = shouldTag ? untagArrayPtr(mergePointers(ptr, cagedPtr), size) : cagedPtr;
-        return untaggedPtr;
+        return Gigacage::caged(kind, ptr);
     }
 
-    T* getMayBeNull(size_t size) const
+    T* getMayBeNull(size_t) const
```

Comment:
Ditto, get rid of the unused `size` argument to reduce client site code.

Reply:
Removed.